apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx

```diff
@@ -131,9 +139,11 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
 
     const [ initialConnectorValues, setInitialConnectorValues ]
         = useState<LoginAttemptSecurityFormInitialValuesInterface>(undefined);
+    const [ manageNotificationInternally, setManageNotificationInternally ] = useState<boolean>(undefined);
```

Comment:
What does `manageNotificationsInternally` mean? which notification will be internally managed when this is set to true?

It feels like the variable name is too generic here. Shall we consider using a better name?